scripts/thredds/README.md

```diff
@@ -0,0 +1,3 @@
+The script in this folder will create a catalog.xml output that can be
+used by the thredds server. This script is run every day and will update
+the thredds server to show the latest data available.
```

Comment:
?example useage (could you paste the crontab here?)

Link to live website as example

Is there a Dockerfile for the server?

Reply:
Link to data use policy here, and to the published webpage